v22.2/cdc-transformations.md

```diff
@@ -0,0 +1,311 @@
+- Stabilize or customize the schema of your changefeed messages for increased compatibility with external systems.
+
+You can use any CockroachDB-supported SQL expression syntax that is not listed in [limitations](#limitations) to build a changefeed query.
+
+See the [Examples](#examples) section for further use cases.
+
+## Syntax
+
+There are two possible components to CDC transformations:
+
+- _Projections_ will select the columns that you want to emit data from. 
+- _Predicates_ will restrict the selected column change data based on the filters you apply.
+
+~~~ sql
+CREATE CHANGEFEED [INTO sink] [WITH options] AS SELECT projection FROM table [WHERE predicate];
+~~~
+
+Parameter        | Description
+-----------------+---------------
+`sink`             | Specify the sink URL to emit change data to. See [Changefeed Sinks](changefeed-sinks.html) for a list of supported sinks. It is also possible to run a changefeed without a sink `CREATE CHANGEFEED WITH...`, which will send changes to the active SQL session.
+`options`          | Set options on the changefeed. See the [Options](create-changefeed.html#options) table for a full list.
+`projection`       | Select the columns from which to emit data.
+`table`            | Define the table to which the columns belong.
+`predicate`        | Apply optional filters with a `WHERE` clause.
+
+For a SQL diagram of the CDC transformation syntax, see the [`CREATE CHANGEFEED`](create-changefeed.html#synopsis) page.
+
+## Limitations
+
+{% include {{ page.version.version }}/known-limitations/cdc-transformations.md %}
+
+## CDC transformation function support
+
+You can use the following functions in CDC transformation queries:
+
+- "Immutable" as marked on the [Functions and Operators page](functions-and-operators.html).
+- Changefeed functions:
+
+    {{site.data.alerts.callout_info}}
+    Changefeed functions are in preview and **subject to change** with continued development.
+    {{site.data.alerts.end}}
+
+    Function                  | Description
+    --------------------------+----------------------
+    `cdc_is_delete() `        | Returns `true` if the event is a deletion event.
+    `cdc_prev()`              | Returns a JSON representation of a row's previous state. 
+    `cdc_updated_timestamp()` | Returns the event's update timestamp. This is typically the MVCC timestamp, but can differ. For example, if the table is undergoing [schema changes](online-schema-changes.html).
+
+- The following "Stable" functions:
+  - `jsonb_build_array()`
+  - `jsonb_build_object()`
+  - `to_json()`
+  - `to_jsonb()`
+  - `row_to_json()`
+  - `overlaps()`
+  - `pg_collation_for()`
+  - `pg_typeof()`
+  - `quote_literal()`
+  - `quote_nullable()`
+
+### Unsupported functions
+
+You can **not** use the following functions with CDC transformations:
+
+- Functions marked as "Volatile" on the [Functions and Operators page](functions-and-operators.html).
+- Functions listed in [Limitations](#limitations) on this page.
```

Comment:
nit: recommend either `listed under the [Limitations] section ...` or `listed in the [Limitations] section ...`